### PR TITLE
docs(eps): update resource support list

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:
update resource support list
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx
supplement some resources support list
**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/data-sources/enterprise_project.md [hunk 1]
```diff
@@ -19,33 +19,68 @@ data "huaweicloud_enterprise_project" "test" {
...
+CSS | huaweicloud_css_cluster |
+DataArts | huaweicloud_dataarts_studio_instance |
+DBSS | huaweicloud_dbss_instance |
+DC  | huaweicloud_dc_virtual_gateway<br>huaweicloud_dc_virtual_interface |
 DCS | huaweicloud_dcs_instance |
-DDS | huaweicloud_dds_instance |
-DMS | huaweicloud_dms_kafka_instance<br>huaweicloud_dms_rabbitmq_instance |
+DDM | huaweicloud_ddm_instance |
+DDS | huaweicloud_dds_instance_v3 |
+Deprecated | huaweicloud_ecs_instance_v1 |
```
Comment (ShiChangkuo) [inline docs/data-sources/enterprise_project.md]: the deprecated resource and the resource with **_v**[1-9] suffix should not be listed

File: docs/data-sources/enterprise_project.md [hunk 2]
```diff
@@ -19,33 +19,68 @@ data "huaweicloud_enterprise_project" "test" {
...
-VPC | huaweicloud_vpc<br>huaweicloud_networking_secgroup | huaweicloud_vpc_subnet<br>huaweicloud_vpc_route<br>huaweicloud_networking_secgroup_rule
+SMS | huaweicloud_sms_task |
+VPC | huaweicloud_vpc<br>huaweicloud_networking_secgroup<br>huaweicloud_vpc_address_group<br> | huaweicloud_vpc_subnet<br>huaweicloud_vpc_route<br>huaweicloud_networking_secgroup_rule
+VPN | huaweicloud_vpn_gateway<br>huaweicloud_vpn_connection |
+WAF | huaweicloud_waf_address_group<br>huaweicloud_waf_certificate<br>huaweicloud_waf_cloud_instance<br>huaweicloud_waf_dedicated_domain<br>huaweicloud_waf_dedicated_instance<br>huaweicloud_waf_domain<br>huaweicloud_waf_policy<br>huaweicloud_waf_reference_table<br>huaweicloud_waf_rule_anti_crawler<br>huaweicloud_waf_rule_blacklist<br>huaweicloud_waf_rule_cc_protection<br>huaweicloud_waf_rule_data_masking<br>huaweicloud_waf_rule_geolocation_access_control<br>huaweicloud_waf_rule_global_protection_whitelist<br>huaweicloud_waf_rule_information_leakage_prevention<br>huaweicloud_waf_rule_known_attack_source<br>huaweicloud_waf_rule_precise_protection<br>huaweicloud_waf_rule_web_tamper_protection |
+WorkSpace | huaweicloud_workspace_desktop<br> |
```
Comment (Lance52259) [inline docs/data-sources/enterprise_project.md]: all rule resources are sub resources

File: docs/data-sources/enterprise_project.md [hunk 3]
```diff
@@ -19,33 +19,68 @@ data "huaweicloud_enterprise_project" "test" {
...
+CSE | huaweicloud_cse__microservice_engine |
+CSS | huaweicloud_css_cluster |
+DataArts | huaweicloud_dataarts_studio_instance |
+DBSS | huaweicloud_dbss_instance |
+DC  | huaweicloud_dc_virtual_gateway<br>huaweicloud_dc_virtual_interface |
 DCS | huaweicloud_dcs_instance |
-DDS | huaweicloud_dds_instance |
-DMS | huaweicloud_dms_kafka_instance<br>huaweicloud_dms_rabbitmq_instance |
+DDM | huaweicloud_ddm_instance |
+DDS | huaweicloud_dds_instance_v3 |
```
Comment (Lance52259) [inline docs/data-sources/enterprise_project.md]:
```suggestion
DDS | huaweicloud_dds_instance |
```

File: docs/data-sources/enterprise_project.md (hunk 1 above)
Comment (ruwenqiang123) [reply docs/data-sources/enterprise_project.md]: have changed

File: docs/data-sources/enterprise_project.md (hunk 2 above)
Comment (ruwenqiang123) [reply docs/data-sources/enterprise_project.md]: have changed

File: docs/data-sources/enterprise_project.md (hunk 3 above)
Comment (ruwenqiang123) [reply docs/data-sources/enterprise_project.md]: have changed

**Pull request merged**